File: server/const/Station.js
class Station {
    constructor(code, name, chiName, line, chiLine) {
      this.code = code;
      this.name = name;
      this.chiName = chiName;
      this.line = line;
      this.chiLine = chiLine;
    }

    getCode() {
      return this.code;
    }

    getName() {
      return this.name;
    }
  
    getChiName() {
      return this.chiName;
    }
  
    getLine() {
      return this.line;
    }
  
    getChiLine() {
      return this.chiLine;
    }
  }

  module.exports = Station;

File: const/Station.py
class Station:
    def __init__(self, code, name, chiName, line, chiLine):
        self.code = code
        self.name = name
        self.chiName = chiName
        self.line = line
        self.chiLine = chiLine

    def getCode(self):
        return self.code

    def getName(self):
        return self.name    

    def getChiName(self):
        return self.chiName

    def getLine(self):
        return self.line

    def getChiLine(self):
        return self.chiLine